flow/project.py

```diff
@@ -791,10 +800,16 @@ def unpack_conditions(condition_functions):
             only returned at the end."""
             callbacks = set()
             for cf in condition_functions:
-                if cf.__name__ == "_flow_metacondition":
+                # condition may not have __name__ attribute
```

Comment:
Is it worth mentioning an example when the `__name__` attribute might be missing? Otherwise I wouldn't know under what circumstances this code path is triggered.